frontend/public/components/utils/kebab.tsx

```diff
@@ -477,14 +477,20 @@ export const ResourceKebab = connectToModel((props: ResourceKebabProps) => {
     actions.map((a) => a(kindObj, resource, null, customData)),
     'hidden',
   );
+  let msg = '';
+  if (_.get(resource.metadata, 'deletionTimestamp')) {
+    msg = terminatingTooltip || t('public~Resource is being deleted');
+  }
   return (
     <Kebab
       options={options}
       key={resource.metadata.uid}
       isDisabled={
-        isDisabled !== undefined ? isDisabled : _.get(resource.metadata, 'deletionTimestamp')
+        // user can set isDisabled to true to disable kebab, but the terminating
+        // message should only be shown if the resource is actually terminating
+        isDisabled !== undefined ? isDisabled : !!_.get(resource.metadata, 'deletionTimestamp')
       }
-      hoverMessage={hoverMessage ? hoverMessage : t('public~Resource is being deleted.')}
+      terminatingTooltip={msg}
```

Comment:
I'm still unclear about this change. Won't this clear out a bunch of existing messages that were previously displayed?

Reply:
By existing, do you mean the older `hoverMessage` usages across the codebase? If so, there's just one, in BlockPools, that I added when I introduced the `hoverMessage` parameter on Kebab.